libraries/GCS_MAVLink/GCS.h

```diff
@@ -613,6 +614,84 @@ class GCS_MAVLINK
 
     uint8_t send_parameter_async_replies();
 
+#if HAVE_FILESYSTEM_SUPPORT
```

Comment:
Should this be gated behind a `#if MAVLINK_FTP_SUPPORT` instead?

Should it be gated on minimise features as well?

Reply:
Discussed with @tridge originally and the consensus was to include it on the minimized builds as well actually as a generally useful feature, and because we want to do some stuff with AP_ROMFS being exposed.

Given that I'd prefer to not introduce more general flags to control this kind of thing.